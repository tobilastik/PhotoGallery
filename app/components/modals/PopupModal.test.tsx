import {fireEvent, render} from '@testing-library/react-native';
import React from 'react';
import {View} from 'react-native';

import {PopupModal, PopupModalProps} from './PopupModal';

const mockProps: PopupModalProps = {
  requestClose: jest.fn(),
  children: <View />,
  modalVisible: false,
};

const renderComponent = () => {
  return render(<PopupModal {...mockProps} />);
};

describe('PopupModal', () => {
  afterEach(jest.clearAllMocks);
  it('renders correctly', () => {
    const {toJSON} = renderComponent();
    expect(toJSON()).toMatchSnapshot();
  });
});
