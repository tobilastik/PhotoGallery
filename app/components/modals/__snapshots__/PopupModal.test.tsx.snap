// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PopupModal renders correctly 1`] = `
<Modal
  hardwareAccelerated={false}
  onRequestClose={[MockFunction]}
  transparent={true}
  visible={false}
/>
`;
